lib/active_model_serializers/adapter/base.rb

```diff
@@ -1,6 +1,9 @@
+require 'active_model_serializers/key_transform'
+
 module ActiveModelSerializers
   module Adapter
     class Base
+      include ActiveModelSerializers::KeyTransform
```

Comment:
`include` can now be removed as we're calling the function on KeyTransform directly